.github/workflows/main.yaml

```diff
@@ -391,4 +392,63 @@ jobs:
           name: ${{ matrix.container-runtime }}-e2e-test-logs
           path: |
             /tmp/cloudcore.log
-            /tmp/edgecore.log
\ No newline at end of file
+            /tmp/edgecore.log
+
+  conformance_e2e_test:
+      runs-on: ubuntu-22.04
+      timeout-minutes: 60
+      name: conformance e2e test
+      needs: image-prepare
+      env:
+        GO111MODULE: on
+      steps:
+        - name: Install Go
+          uses: actions/setup-go@v3
+          with:
+            go-version: 1.20.x
+
+        - uses: actions/cache@v3
+          with:
+            path: ~/go/pkg/mod
+            key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+        - name: Install dependences
+          run: |
+            command -v ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo@${{ env.GINKGO_VERSION }}
```

Comment:
it might not need install ginkgo. suppose it is pre-installed in the container.

Reply:
done